pkg/apis/extensions/types.go

```diff
@@ -937,9 +937,9 @@ type PodSecurityPolicySpec struct {
 	// +optional
 	DefaultAllowPrivilegeEscalation *bool
 	// AllowPrivilegeEscalation determines if a pod can request to allow
-	// privilege escalation.
+	// privilege escalation. If unspecified, defaults to true.
 	// +optional
-	AllowPrivilegeEscalation bool
+	AllowPrivilegeEscalation *bool
```

Comment:
Can you leave it bool on the internal type?

Reply:
since we're defaulting... maybe... will try

Reply:
See:

https://github.com/kubernetes/kubernetes/blob/6039c79f6c583084a2959807f516490fe7232840/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/conversion.go#L141-L148

Reply:
yeah, updated